libsolidity/CompilerUtils.h

```diff
@@ -30,6 +30,9 @@ namespace solidity {
 
 class Type; // forward
 
+/// The size in bytes of the function (hash) identifier
+static const unsigned int g_functionIdentifierOffset = 4;
```

Comment:
you will see what I mean with my next commit. If there is a better way to do it inside the class tell me.